python-sdk/tests/test_storage.py

```diff
@@ -169,3 +169,33 @@ def test_assemble_fileset_incomplete_cache(
                 fetch_missing_chunks=False,
                 http_host_replacement="localhost:9000",
             )
+
+    @staticmethod
+    def test_multichunk_file(client: TestClient, tmp_path: Path):
+        """Test that multiple chunks are retrieved for sufficiently large file."""
+        # Arrange
+        request_iterator = (
+            storage.CreateFilesetRequest(
+                append_file=storage.AppendFile(
+                    path="/bigfile",
+                    data=os.urandom(15 * 1024 * 1024),  # Chunks should max at 20MB
```

Comment:
isn't this 15 MB, which is less than the 20 MB max chunk size? what am I missing here?

Reply:
Sorry for the late response, pachd can only handle gRPC messages up to 20MB in size, so setting to 15MB keeps the message size safely below the limit. This operation appends 15MB to the same file, creating a 30MB file in the fileset.

Reply:
ah gotcha, i missed the for loop